### PR TITLE
Assigments for review

### PR DESCRIPTION
**Pull request by galeal2**

### CODE REVIEW

File: assigment_1_Q1.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+def check_permutation(str1, str2):
+    return sorted(str1.lower())==sorted(str2.lower())
```
Comment (ariava) [inline assigment_1_Q1.py]: Very pythonic :)

Comment (ariava) [inline assigment_1_Q1.py:2]: This file is missing all sorts of testing. Please include a function that performs tests.

File: assigment_1_Q2.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    ll.add(12)
+    ll.add(1)
+    ll.add(2)
+    ll.add(3)
+    ll.add(0)
+    ll.add(5)
+    print(ll)
+    print(find_kth_to_last(ll, 100))
+    print(find_kth_to_last(ll2, 2))
+    print(find_kth_to_last(ll, 3))
```
Comment (ariava) [inline assigment_1_Q2.py:95]: You can use "assert" to perform tests in an automated way (see e.g. https://www.tutorialspoint.com/python/assertions_in_python.htm). The assert statement breaks the program execution with an AssertionError if any of the checks fail.

File: assigment_1_Q1.py (hunk 1 above)
Comment (ariava) [inline assigment_1_Q1.py:2]: This is very pythonic, nice :)

**Review by ariava — CHANGES_REQUESTED** [review]
Please add tests to the Q1 file.

File: assigment_1_Q1.py [hunk 3]
```diff
@@ -1,3 +1,8 @@
 def check_permutation(str1, str2):
     return sorted(str1.lower())==sorted(str2.lower())
 
+#tests
+print(check_permutation('Listen', 'Silent'))
+print(check_permutation('pants', 'stamp'))
+print(check_permutation('2', 'He21'))
+print(check_permutation('cat', ''))
```
Comment (ariava) [inline assigment_1_Q1.py:8]: Cool! Next time try to use "assert" to automate your tests:
e.g.

assert check_permutation('Listen', 'Silent')
assert not check_permutation('2', 'He21')

**Review by ariava — APPROVED** [review]
Thanks for adding tests! Next time try to use assert to automate your tests.

File: assigment_1_Q1.py (hunk 3 above)
Comment (galeal2) [reply assigment_1_Q1.py:8]: Ok,
Thank you!

**Pull request merged**